CDCgov/prime-reportstream
# Cle/bundle destination

This PR moves the hard coded phase 1 routing from FHIRTranslator to FHIRRouter, gets the 'target' elements into the bundle, and adds tests

Test Steps:
1. Run automated tests
OR
1. Run a message through universal pipeline 
2. Verify that a 'send' message for co-phd.full-elr-hl7' is generated

## Changes
- put targets into bundle in the FHIRRouter
- pull targets out of bundle in the FHIRTranslator
- add tests

## Checklist

### Testing

Resolves #7035
